src/apis/line.test.ts

```diff
@@ -48,14 +48,11 @@ test("get user profile", async () => {
 });
 
 test("LINE Login authorize page", async () => {
-  const { response } = await api.GET("/line-login/oauth2/v2.1/authorize", {
-    params: {
-      query: {
-        client_id: "test_client",
-        redirect_uri: "http://example.com/callback",
-      },
-    },
-  });
+  const url = new URL("/line-login/oauth2/v2.1/authorize", baseUrl);
+  url.searchParams.set("client_id", "test_client");
+  url.searchParams.set("redirect_uri", "http://example.com/callback");
+  
+  const response = await fetch(url.toString());
```

Comment:
No need to call fetch directly - we have `apiFetch` which is a configured `ofetch` instance.